From abc95d236dd50dbd0b7baf018c412b49a59d24a2 Mon Sep 17 00:00:00 2001
From: Ashlesh Gawande <agawande@memphis.edu>
Date: Thu, 21 Feb 2019 16:58:26 -0600
Subject: [PATCH] wscript: changes for android build

---
 wscript | 13 ++++++++-----
 1 file changed, 8 insertions(+), 5 deletions(-)

diff --git a/wscript b/wscript
index 1307138..f0ca601 100644
--- a/wscript
+++ b/wscript
@@ -24,11 +24,9 @@ def configure(conf):
                'default-compiler-flags', 'boost',
                'doxygen', 'sphinx_build'])
 
-    if 'PKG_CONFIG_PATH' not in os.environ:
-        os.environ['PKG_CONFIG_PATH'] = Utils.subst_vars('${LIBDIR}/pkgconfig', conf.env)
-
-    conf.check_cfg(package='libndn-cxx', args=['--cflags', '--libs'],
-                   uselib_store='NDN_CXX', mandatory=True)
+    conf.add_os_flags("LIB_NDN_CXX")
+    conf.add_os_flags("CXXFLAGS_NDN_CXX")
+    conf.add_os_flags("LINKFLAGS_NDN_CXX")
 
     conf.env['WITH_TESTS'] = conf.options.with_tests
     conf.env['WITH_EXAMPLES'] = conf.options.with_examples
@@ -54,7 +52,12 @@ def configure(conf):
     conf.write_config_header('PSync/detail/config.hpp')
 
 def build(bld):
+    version(bld)
+
     bld.shlib(
+        name='PSync',
+        vnum=VERSION_BASE,
+        cnum=VERSION_BASE,
         target='PSync',
         source =  bld.path.ant_glob('PSync/**/*.cpp'),
         use = 'BOOST NDN_CXX',
-- 
2.17.1

